port/zos390/omrsl.c

```diff
@@ -287,6 +287,9 @@ omrsl_lookup_name(struct OMRPortLibrary *portLibrary, uintptr_t descriptor, char
 	{
 		handle = (dllhandle *)descriptor;
 		address = (void *)dllqueryfn(handle, name);
+		if (NULL == address) {
+			address = (void *)dllqueryvar(handle, name);
```

Comment:
I think we should avoid doing this when it's not needed i.e. for jdk8 - jdjk17.
I'm not sure if there is a way to query the java version. If nothing else there is portControl function that can be used to pass in hints to control port library behavior.